boxoffice/views/admin_discount.py

```diff
@@ -52,12 +62,38 @@ def jsonify_discount_policies(data_dict):
     )
 @render_with({'text/html': 'index.html', 'application/json': jsonify_discount_policies}, json=True)
 def admin_discount_policies(organization):
+    RESULTS_PER_PAGE = 6
     query = request.args.get('search')
-    if query:
-        discount_policies = DiscountPolicy.query.filter(DiscountPolicy.title.ilike('%{query}%'.format(query=query))).all()
+    try:
+        page = int(request.args.get('page', 1))
+    except ValueError:
```

Comment:
I meant if we could use an `if/else` instead of using a `try/catch`. i.e

```python
if request.args.get('page'):
    page = int(request.args.get('page'))
else:
    page = 1
```